clangify.sh

```diff
@@ -1,18 +1,22 @@
 #!/bin/bash
 
-# This script will run clang-format on all non-3rd-party C++/Header files.
+# This script will run clang-format on all modified, non-3rd-party, C++/Header files.
```

Comment:
It's an enumeration of two adjectives. It should be ok with no or one comma, but not two.